### PR TITLE
Popup details Window 

### PR DESCRIPTION
**Pull request by Mehdi-Rh**
In this PR, we have added popup windows for the work section for the mobile and the desktop version that fit the Figma template.

### Main changes 

- Add a work-popup.css and a work-popup.js file 
- Make popup windows for the works section with the following features:

  - When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
  - When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: styles/work-popup.css [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+  border-radius: 10px;
+  filter: blur(0);
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: row;
+}
+
+.modal-content {
```
Comment (descholar-ceo) [inline styles/work-popup.css:20]: - [ ] There are still issues with responsiveness: 
  - [ ] The text in `See Source` button is overlapping the button when resized the screen from the large screens to small screens

File: styles/work-popup.css [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+  justify-content: center;
+  background-color: rgba(255, 255, 255, 0.1);
+  backdrop-filter: blur(10px);
+  width: 85%;
+  margin: 0 auto;
+}
+
+.modal-head {
```
Comment (descholar-ceo) [inline styles/work-popup.css:28]: - [ ] When we click the `See project` button in small screens, the bottom and header of the popup box aren't visible, hence navigating through the page is difficult, would you address this issue?
 - [ ] The images are stretched in small screens, which gives a bad user experience

Kindly watch the next screencast to understand more about the context of what I am talking about

https://user-images.githubusercontent.com/56886064/151597156-3bec2366-d30b-4b7f-a35e-fc5ebd6cf5a3.mp4

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
Hi @Mehdi-Rh , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: work-popup.js [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+    </ul>
+    <div class="modal-image">
+      <img src=${projects[cardNbr].src} alt="App Image">
+    </div>
+    <div class="modal-description">
+      <p>${projects[cardNbr].description[0]}</p>
+      <p>${projects[cardNbr].description[1]}</p>
+    </div>
+    <div class="modal-technologies">
+      <ul class="tag_list no_bullet">
```
Comment (brytebee) [inline work-popup.js:105]: - [ ] **Required:** On the mobile popup, kindly fix the following:
- Some description are undefined
- Add horizontal rule `hr`
- Properly align buttons

Your Implementation | Figma Template
:---: | :---:
image | image

- [ ] **Required:** Kindly align text to the left.

Your Implementation | Figma Template
:---: | :---:
image | image

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Mehdi-Rh 👋, 

Great work on making the changes 👏 👏 
You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: work-popup.js (hunk 3 above)
Comment (oloomoses) [reply work-popup.js:105]: - [ ] Some of the changes suggested by @brytebee in the previous review are not yet implemented. Kindly match the popup with the given design by fixing text alignment and buttons as suggested by @brytebee with the screenshots he provided.

**Review by oloomoses — CHANGES_REQUESTED** [review]
Hi @Mehdi-Rh, 

You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**